### ISSUE TITLE
Keyboard focus lost after invoking hamburger menu at resized resolution: A11y_Playwright OSS releases_Installation_Resize

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/playwright.dev#1851]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#Playwright OSS Releases;#BM_PlaywrightOSSReleases_Web_Aug2025;#Chrome;#DesktopWeb;#FTP;#Win11;#A11ySev2;#A11yMAS;#WCAG1.4.4;#Resize;#Zoom;

### Environmental Details:
App Name: Playwright OSS releases
#URL: https://playwright.dev/docs/intro
OS & Browser Details:
Microsoft Windows 11 Enterprise
Version 26100.4946
Chrome
Version 139.0.7258.139 (Official Build) (64-bit)

### Prerequisites to apply resize:
Change system resolution to 1280*768 and zoom browser size to 200%.

### Repro Steps:

1. Hit the URL: https://playwright.dev/docs/intro.
2. Installation page will get open.
3. Observe that Keyboard focus lost after invoking hamburger menu at resized resolution or not.

### Actual:
After invoking the hamburger menu at resized resolution, keyboard focus becomes lost and is no longer visible.
Users relying on keyboard navigation cannot determine where the focus is, preventing further interaction.

### Expected:
 After invoking the hamburger menu at resized resolution, keyboard focus should be visible and move logically through all interactive elements.

### User Impact:
Users with visual or motor impairments relying on keyboard navigation may become disoriented.

### Attachment:

https://github.com/user-attachments/assets/1eeccc83-06f4-46ff-9cd1-1efbfac54a2f

**Comment by agg23** [comment]
This is a Docusaurus issue that is unlikely to be solved. Closing
